src/templates/components/imageAndStaticMap/index.tsx

```diff
@@ -14,18 +13,7 @@ export const ImageAndStaticMap: React.FC<ImageAndStaticMapProps> = ({
 }) => {
   return (
     <div className="usa-width-one-third inline-table-helper vads-u-order--first mobile-lg:vads-u-order--initial vads-u-margin-bottom--2 vads-u-margin-left--auto facility">
-      <div
-        onClick={() =>
-          recordEvent({ event: 'image-click', 'facility-name': facilityId })
-        }
-        style={{ cursor: 'pointer' }}
-      >
```

Comment:
I'm not sure why we were recording click events to a non-interactive image. This comes from `content-build`, but I think from a misunderstanding of what that image template was actually doing. That image template was also used to render images inside `<a>` tags. The click event would pass through both the `<a>`'s `onClick` handler _and_ the `<img>`'s `onClick` handler. This...isn't needed right here. 🤷

Reply:
Nice! I like that we are clearing out a lot of this tech debt stuff